Gravity/Gravity/Extensions/RdoExtensions.cs

```diff
@@ -31,7 +31,10 @@ public static T ToHydratedDto<T>(this RDO rdo)
 
 					switch (fieldAttribute.FieldType)
 					{
-						case RdoFieldType.Currency:
+					    case RdoFieldType.SingleObject:
```

Comment:
Try and be careful mixing whitespaces and tabs. You may want to install this: https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.FixMixedTabs

Reply:
good catch, I think I fixed will look at the plugin.